config/core-site.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration>
+
+<!-- WASB (Azure Storage) Configuration  -->
+<!-- More information about WASB can be foudn at https://docs.microsoft.com/azure/hdinsight/hdinsight-hadoop-use-blob-storage> -->
+<!-- <property>
+  <name>fs.AbstractFileSystem.wasb.Impl</name>
```

Comment:
I see its 2 space for this file should probably add .xml to the list of extensions that should have 2 space indent(in `.editorconfig`)